base/collect_grpc.go

```diff
@@ -11,7 +11,7 @@ import (
 
 const (
 	// CollectGRPCTaskName is the name of this task which performs load generation and metrics collection for gRPC services.
-	CollectGRPCTaskName = "gen-load-and-collect-metrics-grpc"
+	CollectGRPCTaskName = "grpc"
```

Comment:
Short names are easier on the documentation. Task template `_task-grpc`, task name `grpc`, chart values `.Values.grpc.call`, etc., can also be made concise and consistent with short names.